client.go

```diff
@@ -205,6 +212,26 @@ func (c *Client) do(ctx context.Context, method string, path string, query Query
 	return
 }
 
+func parseRespMeta(h http.Header) responseMeta {
+	var ret responseMeta
+	if v, ok := h["X-Consul-KnownLeader"]; ok && len(v) > 0 {
+		ret.knownLeader = true
+	}
+	if v, ok := h["X-Consul-Translate-Addresses"]; ok && len(v) > 0 {
+		ret.translateAddresses = true
+	}
+	ret.index, _ = strconv.ParseUint(h.Get("X-Consul-Index"), 10, 64)
+	ret.lastContact, _ = strconv.ParseUint(h.Get("X-Consul-LastContact"), 10, 64)
```

Comment:
Is it documented that this is an unsigned value? I wonder if we should use negatives as a way to indicate that the value wasn't found (unless zero is not a valid index/last-contact, in which case it can be used as the _missing_ marker).

Reply:
I pulled this from the server source.  There's not a whole lot of docs about it.   I haven't seen it set to anything but 0 so far though.

Index should always be increasing and never be negative.

Open to alternatives.  I don't like ignoring the error but not sure what the caller would do with the error.

Reply:
If `index` and `lastContext` aren't supposed to be negative it seems like a good way to report that the value was missing (assuming there's no documentation saying that it cannot be 0).